src/Microsoft.Data.SqlClient/src/Interop/Windows/Sni/SniNativeMethodsArm64.netfx.cs

```diff
@@ -0,0 +1,319 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if NETFRAMEWORK
+
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Data.SqlClient;
+
+namespace Interop.Windows.Sni
+{
+    internal class SniNativeMethodsArm64 : ISniNativeMethods
```

Comment:
```suggestion
    internal sealed class SniNativeMethodsArm64 : ISniNativeMethods
```

A few other cases of this too.

Reply:
Yep, good call. I did it later, but it makes sense to do it now.